grakn-kb/src/main/java/ai/grakn/kb/internal/GraknTxAbstract.java

```diff
@@ -109,6 +111,7 @@
 public abstract class GraknTxAbstract<G extends Graph> implements GraknTx, GraknAdmin {
     final Logger LOG = LoggerFactory.getLogger(GraknTxAbstract.class);
     private static final String QUERY_BUILDER_CLASS_NAME = "ai.grakn.graql.internal.query.QueryBuilderImpl";
+    private static final String QUERY_RUNNER_CLASS_NAME = "ai.grakn.graql.internal.query.runner.TinkerQueryRunner";
```

Comment:
`TinkerQueryRunner.getClass().getName()`. In my opinion they will be more refactor-proof

Reply:
Ditto above.